src/migrate-component/component-info-utils.ts

```diff
@@ -179,3 +182,23 @@ const findTemplateUrl = (componentPath: string, componentClassName: string, tree
   }
 }
 
+const findStyleUrl = (componentPath: string, componentClassName: string, tree: Tree): string => {
+  const source = getSourceFile(tree, componentPath);
+
+  const node = findDecoratorPropertyNode(source, componentClassName, 'Component', 'styleUrls');
+  if (node === null) {
+    return '';
+  }
+
+  if (ts.isArrayLiteralExpression(node) && node.elements.length > 0) {
+    const stylePath = (node.elements[0] as ts.StringLiteral).text;
```

Comment:
What if there are multiple stylesheets for the same component?

Reply:
I thought about that, but 99.9% of the cases would use just a single stylesheet. And in the cases that there are more it would probably be better to handle it manually